orchestrator/channels/base.py

```
@@ -0,0 +1,112 @@
+                if reply_channel:
+                    await self.send_message(reply_channel, decision.response)
+
+        except Exception as e:
+            logger.error(
+                "[Channel:%s] Failed to handle message: %s",
+                self.connection_id,
+                e,
+            )
+
+    @abstractmethod
+    def _to_envelope(self, platform_message: Dict[str, Any]) -> Optional["RequestEnvelope"]:
```

Ruff (0.15.0)

[error] Undefined name `RequestEnvelope`

(F821)